### PR TITLE
Overdose Tweaks

### PR DESCRIPTION
**Pull request by Lohikar**
Partially reverts OD changes, as per skull's request:
> Let's do this:
>
>    Remove OD from chemicals where they already fix toxins damage. So dylovene, tricord, etc. Also from Bicard, because of point two.
>    Flip the odd damage types for kelotane, dexalin, dermaline to toxins and raise them to higher counts. Like 40 units (as is already the case with spaceacillin).
>
>That should be a workable solution until we actually implement something deeper.

### CODE REVIEW

**Comment by skull132** [comment]
What is: label hell.

**Comment by skull132** [comment]
Also, the reason why the way OD is counted was changed is to deal better with slow metabo drugs. Specifically spaceacillin and co, which have OD effects come into play a legitimate 5 minutes after OD. Any thoughts on how to handle that?

**Comment by Lohikar** [comment]
Overdoses should apply at the same time as the effects, so I don't see much reason to make those chems OD faster or something.

**Comment by skull132** [comment]
Let's do this:
* Remove OD from chemicals where they already fix toxins damage. So dylovene, tricord, etc. Also from Bicard, because of point two.
* Flip the odd damage types for kelotane, dexalin, dermaline to toxins and raise them to higher counts. Like 40 units (as is already the case with spaceacillin).

That should be a workable solution until we actually implement something deeper.

**Comment by Lohikar** [comment]
whoops, forgot to remove DNM.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 1]
```diff
@@ -166,11 +151,6 @@
 		M.heal_organ_damage(3 * removed, 3 * removed)
 		M.adjustToxLoss(-3 * removed)
 
-/datum/reagent/tricordrazine/overdose(var/mob/living/carbon/M, var/alien)
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Inconsistent.
Tricrod has no effect on affect_blood on diona, but defaults to the standoard overdose effect, which effects diona.

Comment (Arrow768) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: The same applies to a few others (dermaline, kelotane, dexalin are the ones I could find quickly)
All those do not affect diona in affect_blood, but affect diona in overdose

**Comment by Lohikar** [comment]
Replaced by #3499.

**Pull request closed without merging**